test/ut3_user/expectations/binary/test_to_be_within.pks

```diff
@@ -0,0 +1,31 @@
+create or replace package test_to_be_within is
```

Comment:
Tests for `not_to_be_within` are missing and so we can't say if the behavior works ok for negated matcher.